IO2P/resourceAdder.cs

```diff
@@ -8,14 +8,18 @@
 using MongoDB.Driver;
 using System.Net;
 using System.IO;
+using System.Net.Sockets;
+using System.Net.Http;
+using System.Web;
 
 namespace IO2P
 {
     class resourceAdder
     {
-        private String strDatabase = "";
-        private String strDBUser = "";
-        private String strDBPass = "";
+        private String host = "146.185.128.224:27017"
+        private String strDatabase = "io2";
+        private String strDBUser = "io2";
+        private String strDBPass = "sfd9879IjgkDslkhgdgl98sdfG9sdUFSD98sdf9wfdgHG78vgsf809few0";
```

Comment:
Hosta też dajmy. Powiedzmy DB_HOST. W zasadzie port też można dać konfigurowalny jako DB_PORT. Travis będzie miał bazę lokalnie (127.0.0.1) więc na pewno musimy mieć to w konfigu.